dvc/command/ls/__init__.py

```diff
@@ -14,15 +17,21 @@ def run(self):
         from dvc.repo import Repo
 
         try:
-            nodes = Repo.ls(
+            fs_nodes = Repo.ls(
                 self.args.url,
                 self.args.target,
                 rev=self.args.rev,
                 recursive=self.args.recursive,
                 outs_only=self.args.outs_only,
             )
-            if nodes:
-                logger.info("\n".join(nodes))
+            if fs_nodes:
+
+                @colorize(with_color=sys.stdout.isatty())
+                def prettify(fs_node):
+                    return str(fs_node["path_info"])
```

Comment:
Single use decorator on a throw away function looks weird. Why not simply combine the two into single function?

Reply:
I would like to keep it as a decorator
1. output changes really often (and with including in future additional info will change also), splitting it by different functions make it more readable
2. I need to initialize `LsColors` object before invocation and decorator helps me to do it the most simple way